.github/actions/spelling/candidate.patterns

```diff
@@ -156,7 +161,7 @@ GHSA(?:-[0-9a-z]{4}){3}
 # GitLab commits
 \bgitlab\.[^/\s"]*/(?:[^/\s"]+/){2}commits?/[0-9a-f]+\b
 
-# binanace
+# binance
```

Comment:
Yeah, I had a typo in my files -- the hazard of excluding them by default -- it took a while for me to catch 😊